op-e2e/sequencer_failover_test.go

```diff
@@ -17,3 +21,99 @@ func TestSequencerFailover_SetupCluster(t *testing.T) {
 		require.NotNil(t, con, "Expected conductor to be non-nil")
 	}
 }
+
+// [Category: conductor rpc]
+// In this test, we test all the rpcs exposed by conductor.
+func TestSequencerFailover_ConductorRPC(t *testing.T) {
+	ctx := context.Background()
+	sys, conductors := setupSequencerFailoverTest(t)
+	defer sys.Close()
+
+	// SequencerHealthy, Leader, AddServerAsVoter are used in setup already.
+
+	// Test ClusterMembership
+	c1 := conductors[Sequencer1Name]
+	c2 := conductors[Sequencer2Name]
+	c3 := conductors[Sequencer3Name]
+	membership, err := c1.client.ClusterMembership(ctx)
+	require.NoError(t, err)
+	require.Equal(t, 3, len(membership), "Expected 3 members in cluster")
+	ids := make([]string, 0)
+	for _, member := range membership {
+		ids = append(ids, member.ID)
+		require.Equal(t, consensus.Voter, member.Suffrage, "Expected all members to be voters")
+	}
+	sort.Strings(ids)
+	require.Equal(t, []string{Sequencer1Name, Sequencer2Name, Sequencer3Name}, ids, "Expected all members to be voters")
```

Comment:
This check isn't quite to "Expect all members to be voters", but rather to expect the 3 sequencers to be included, right? That voter check happens above on line 44?

Reply:
You're correct, the description could've been clearer.

The more accurate description could be `Expect all sequencers to be inside cluster membership`